From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Huang Jianan <huangjianan@oppo.com>
Date: Sat, 20 Feb 2021 10:29:52 +0800
Subject: ANDROID: GKI: enable CONFIG_EROFS_FS=y

CONFIG_EROFS_FS=y is enable in android-5.4-stable branch now, so open
it on mainline to avoid repeated submissions in other branches.

Bug: 180727180
Change-Id: Ifb984d0816c043fca4699226a481b5f8ab7ab6fd
Signed-off-by: Huang Jianan <huangjianan@oppo.com>
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -525,6 +525,7 @@ CONFIG_PSTORE=y
 CONFIG_PSTORE_CONSOLE=y
 CONFIG_PSTORE_PMSG=y
 CONFIG_PSTORE_RAM=y
+CONFIG_EROFS_FS=y
 CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_CODEPAGE_737=y
 CONFIG_NLS_CODEPAGE_775=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -465,6 +465,7 @@ CONFIG_PSTORE=y
 CONFIG_PSTORE_CONSOLE=y
 CONFIG_PSTORE_PMSG=y
 CONFIG_PSTORE_RAM=y
+CONFIG_EROFS_FS=y
 CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_CODEPAGE_737=y
 CONFIG_NLS_CODEPAGE_775=y
